plugins/CoreHome/javascripts/broadcast.js

```diff
@@ -784,10 +784,10 @@ var broadcast = {
 
             // try looking for multi value param
             lookFor = param + '[]=';
-            startStr = url.indexOf(lookFor);
+            startStr = url.lastIndexOf(lookFor);
             if (startStr >= 0) {
                 var result = [getSingleValue(startStr)];
-                while ((startStr = url.indexOf(lookFor, startStr + 1)) !== -1) {
+                while ((startStr = url.lastIndexOf(lookFor, startStr + 1)) !== -1) {
```

Comment:
Would this be an issue if we have an array param like `&key[]=val1&key[]=val2&key[]=val3` where the new array value has reversed order? Eg, w/ lastIndexOf, I would expect it to be `['val3','val2','val1']`, but callers should expect it to be the other way around.

Reply:
I'll revert that, when searching for multiple values